ginkgo.Jenkinsfile

```diff
@@ -53,6 +53,24 @@ pipeline {
                }
             }
         }
+        stage('UnitTesting') {
```

Comment:
Rename this stage to `Privileged Unit Testing` and add a comment in the `Jenkinsfile` indicating why we are running this stage in Jenkins vs. in Travis.

Reply:
This applies more to 
https://github.com/cilium/cilium/pull/6180 - I'll leave this comment on there.